src/Core/Constants.cs

```diff
@@ -116,6 +116,7 @@ public static class FeatureFlagKeys
     public const string ImportAsyncRefactor = "pm-22583-refactor-import-async";
     public const string CreateDefaultLocation = "pm-19467-create-default-location";
     public const string MembersGetEndpointOptimization = "pm-23113-optimize-get-members-endpoint";
+    public const string DirectoryConnectorRemoveUsersFix = "pm-20140-directory-connector-remove-users-fix";
```

Comment:
Will you please update the key to `pm-24592-directory-connector-prevent-user-removal`